printing_templates/One Dive.html

```diff
@@ -111,117 +148,135 @@
 							</tr>
 							<tr>
 								<td class="fieldTitle">
-									<h1> Location </h1>
+									<b> Time </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.location }} </p>
+									<p> {{ dive.time }} </p>
 								</td>
-							</tr>
-							<tr>
 								<td class="fieldTitle">
-									<h1> Max. depth </h1>
+									<b> Gases </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.depth }} </p>
-								</td>
+									<p> {{ dive.gas }} </p>
+								</td>								
 							</tr>
 							<tr>
 								<td class="fieldTitle">
-									<h1> Duration </h1>
+									<b> Location </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.duration }} </p>
+									<p> {{ dive.location }} </p>
 								</td>
-							</tr>
-						</tbody></table>
-						<table class="table_class">
-							<tbody><tr>
 								<td class="fieldTitle">
-									<h1> Time. </h1>
+									<b> Water Temp. </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.time }} </p>
-								</td>
+									<p> {{ dive.waterTemp }} </p>
+								</td>								
 							</tr>
 							<tr>
 								<td class="fieldTitle">
-									<h1> Air Temp. </h1>
+									<b> Max Depth </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.airTemp }} </p>
+									<p> {{ dive.depth }} </p>
 								</td>
-							</tr>
-							<tr>
 								<td class="fieldTitle">
-									<h1> Water Temp. </h1>
+									<b> Buddy </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.waterTemp }} </p>
-								</td>
+									<p> {{ dive.buddy }} </p>
+								</td>								
 							</tr>
 							<tr>
 								<td class="fieldTitle">
-									<h1> Buddy </h1>
+									 <b> Duration </b>
 								</td>
 								<td class="fieldData">
-									<p> {{ dive.buddy }} </p>
+									<p> {{ dive.duration }} </p>
 								</td>
-							</tr>
-							<tr>
 								<td class="fieldTitle">
-									<h1> Divemaster </h1>
+									<b> Dive Master </b>
 								</td>
 								<td class="fieldData">
 									<p> {{ dive.divemaster }} </p>
-								</td>
+								</td>								
 							</tr>
-							</tbody>
 						</table>
+				</div>
+
+
+				<div class="diveProfile" id="dive_{{ dive.id }}">
+
+				</div>
+
+				<div class="notesSection">
 						<table class="notes_table_class">
 							<tbody>
 							<tr>
 								<td class="fieldTitle">
-									<h1> Notes </h1>
+									<b>&nbsp;Notes </b>
 								</td>
 							</tr>
 							<tr>
 								<td class="fieldData">
-									<div class="textArea">
-										<p> {{ dive.notes|safe }} </p>
+									<input type="hidden" id="dive_notes_hidden" name="dive_notes_hidden" value="{{ dive.notes|safe }}" />
+									<div class="textArea" id="dive_notes_display" >
+										
 									</div>
 								</td>
 							</tr>
 							</tbody>
 						</table>
-					</div>
-				</div>
-			</div>
-		</div>
+				</div>  <!-- notesSection -->
+
+<script>
```

Comment:
please, move the `<script>...</script>` under the closing `</style>` tag in the `<head>...</head>` closure.

Reply:
or try moving it to a location outside the `<body>` tag?